docker/linux/Dockerfile

```diff
@@ -183,12 +182,13 @@ ENV PATH="/tools/pinguino-compilers/p32/bin:$PATH"
 # RISCV toolchain
 COPY --from=nuttx-toolchain-riscv /tools/riscv64-unknown-elf-gcc/ riscv64-unknown-elf-gcc/
 ENV PATH="/tools/riscv64-unknown-elf-gcc/bin:$PATH"
- 
+
 # ESP32 toolchain
 COPY --from=nuttx-toolchain-esp32 /tools/xtensa-esp32-elf-gcc/ xtensa-esp32-elf-gcc/
+COPY --from=nuttx-toolchain-esp32 /tools/esp-idf/ esp-idf/
 ENV PATH="/tools/xtensa-esp32-elf-gcc/bin:$PATH"
 RUN mkdir -p /tools/blobs/esp32core
 COPY --from=nuttx-toolchain-esp32 /tools/blobs/* /tools/blobs/esp32core/
 RUN pip3 install esptool
 
-ENTRYPOINT [ "/bin/bash" ]
+CMD [ "/bin/bash" ]
```

Comment:
Why are we changing this?

Reply:
I found an issue when trying to do `docker run .... /bin/bash`, as I mentioned in the relevant commit message. My use case for doing that was piping bash scripts into the `docker run` command. Let me know if this definitely cannot be accomodated.

Reply:
you can always override it with --entrypoint.

on the other hand, the ENTRYPOINT usage here looks a bit unusual for me.
@btashton can you explain why it's done this way? i guess the ci can specify bash, rather than just "-c".

Reply:
I agree that it is odd. I'm not sure why we used entrypoint in the first place. Happy to have it corrected so CMD can be used, we just need to also update this

https://www.github.com/apache/incubator-nuttx-testing/tree/master/.github%2Factions%2Fci-container%2Faction.yaml

Reply:
I have added a commit for this. Let's see if it's good enough...

Reply:
Well, it wasn't. I found a workaround for my use case, so I will revert the github action and the Dockerfile and if needed at some point this topic can be revisited.

Reply:
I just merged it. The issue here is that the CI is brittle and won't use the new container because it has not been published. That only happens once it has been merged in.  We can revert that commit if the master build fails.

Reply:
That's good, thank you.